### PR TITLE
fix: min max range on horizontal bar chart

### PR DESCRIPTION
**Pull request by almeidabbm**
Closes: #9803 

### Description:

Before:

image

After:

image

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by rephus** [comment]
I'm still getting the issue described in https://github.com/lightdash/lightdash/issues/9803

[Screencast from 22-04-24 13:43:42.webm](https://github.com/lightdash/lightdash/assets/1983672/fd26a0d3-cabc-4548-ad0f-1f005f871ce4)

File: packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts [hunk 1]
```diff
@@ -1535,11 +1538,13 @@ const useEchartsCartesianConfig = (
         }
     }, [
         validCartesianConfig?.layout?.xField,
+        validCartesianConfig?.layout.flipAxes,
+        validCartesianConfig?.layout?.yField,
         validCartesianConfig?.eChartsConfig?.series,
         rows,
         itemsMap,
+        axes,
```
Comment (rephus) [inline packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts:1546]: unnecesary field ?

**Comment by almeidabbm** [comment]
> I'm still getting the issue described in #9803
> 
>  Screencast.from.22-04-24.13.43.42.webm

This should be fixed now, the range filter was hiding values that were bigger than the max: [fix commit](https://github.com/lightdash/lightdash/pull/9844/commits/744349f00d8242c29088ca04e261189d3645f84f)

**Comment by rephus** [comment]
> > I'm still getting the issue described in #9803
> > Screencast.from.22-04-24.13.43.42.webm
> 
> This should be fixed now, the range filter was hiding values that were bigger than the max: [fix commit](https://github.com/lightdash/lightdash/pull/9844/commits/744349f00d8242c29088ca04e261189d3645f84f)
> 
> But I now have a question, do we want to keep values greater than max hidden? If so then I believe the prev implementation was correct in removing them from the values

I think

**Comment by TuringLovesDeathMetal** [comment]
I feel like we should keep the bar charts showing? 👀 Even if they exceed the axis limit. 

Ideally, there's some indication that they've exceeded the limit (or an option to include one), but let's not add that for now until someone complains. 

I'd expect the same behaviour for area charts btw!

**Review by rephus — APPROVED** [review]

**Pull request merged**